### ISSUE TITLE
[Bug]: equivalince t-tests not working

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2141]
### JASP Version

Others

### JASP Module

Equivalence T-Tests

### What analysis are you seeing the problem on?

all

### What OS are you seeing the problem on?

Flatpak

### Bug Description

![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/b67fb7da-d543-4e5a-aa90-e335c293b00d)

Only the desciptives work.
Graphs never come up.
And inference only comes up when I choose "bounds specification" "Raw" instead of Cohens d.
for the one sample t-Test also the results do not change if I change the "test value" which seems not okay to me... even if I am new to this kind of test.

To sum up: The complete module seems kinda broken.

### Expected Behaviour

graphs and inference should display 

### Steps to Reproduce

1. open any dataset
2. run a test
3.
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
this is for version 0.17.2 which is currently not selectable in the bug report dropdown

**Comment by koenderks** [comment]
I believe this pr should fix that (https://github.com/jasp-stats/jaspEquivalenceTTests/pull/88), but it needs a review on the failing unit tests from someone with knowledge of this analysis

**Comment by tomtomme** [comment]
The following is still broken in 0.18.2 flatpak-beta from 30.11.23 in the 3 classic equivalence t-Tests

- selecting cohens d as bound specification
- all plots/graphs
- inference of classic paired samples t-Test

**Comment by boutinb** [comment]
The fix was made 2 weeks ago. I have tested it, and the analyses seem to work correctly. I think there is a new flatpak made, maybe you can try this one.

**Comment by tomtomme** [comment]
thx. No new flatpak yet. It usually needs some days to propagate to all mirrors.

**Comment by shun2wang** [comment]
You can try the testing repository:
```
flatpak install --user https://dl.flathub.org/build-repo/71242/org.jaspstats.JASP.flatpakref
```

**Comment by tomtomme** [comment]
Thx @shun2wang 
Will this repo stay the same for future flatpak builds? Is this documented somewhere? Or how can I find out what the current build-number is, assuming 71242 is the build number

**Comment by shun2wang** [comment]
That's on a daily build on flatpak repo, you can find at some build PR. https://github.com/flathub/org.jaspstats.JASP/pull/53

**Comment by tomtomme** [comment]
The following is still broken in 0.18.2 flatpak-beta from 18.12.23 in equivalence t-Tests

- classic paired samples t-Test 
tested with the "moon and aggression" data from the data library.

![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/dd7e3a21-946c-4f8a-a962-307c65c6c770)

**Comment by shun2wang** [comment]
Please note that you may need to select the correct branch to run when running flatpak, otherwise will running default branch (stable) .Did you run the flatpak by right branch? you may run something like this:

```
flatpak run --branch=test org.jaspstats.JASP
```

**Comment by shun2wang** [comment]
and check the commit link in JASP's about page.  paste that link to here1.

**Comment by shun2wang** [comment]
and check the commit link in JASP's about page.  paste that link to here1.

**Comment by tomtomme** [comment]
![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/e10b89c1-0c00-4e82-92f8-64fbc7160276)

no commit link in my about page - but yeah - I am running the up to date beta

**Comment by shun2wang** [comment]
so that problem still on this build?

**Comment by tomtomme** [comment]
source link: https://github.com/jasp-stats/jasp-desktop/commit/083377444ea57668cc56ce42a2acaa6ec4255c6e

commit 0833774

**Comment by boutinb** [comment]
Yes this problem still exists...
Maybe can @vandenman has a look?

**Comment by tomtomme** [comment]
yeah
_paired_ samples t-test (equivalence) still broken
